rabbitmq/README.md

```diff
@@ -0,0 +1,31 @@
+# RabbitMQ Docker files with Alpine Linux
+
+> These images are only for development environments
+
+## Build upon
+
+The [official](https://hub.docker.com/_/rabbitmq/) rabbitmq docker images, specifically alpine with the management plugin enabled.
+
+## What does this image do?
+
+- Configures rabbitmq to use TLS with a self-signed localhost certificate. 
+- Enables the management plugin.
+- Sets a default `prooph` user with password `prooph`.
```

Comment:
It does? I'm seeing it create the user `admin`.

https://github.com/basz/docker-files/blob/9707cb2bfee3210669a1832d7a85cd492244a952/rabbitmq/Dockerfile#L7